functions/Get-DbaConfiguration.ps1

```diff
@@ -0,0 +1,108 @@
+﻿FUNCTION Get-DbaConfiguration
```

Comment:
I feel we need to put "sql" or "instance" somewhere in the name, because the OS and other stuff have configurations too.